lib/madmin/resource.rb

```diff
@@ -48,11 +52,11 @@ def new_path
       end
 
       def show_path(record)
-        "/madmin/#{model.model_name.collection}/#{record.id}"
+        "/madmin/#{model.model_name.collection}/#{friendly_id(record)}"
       end
 
       def edit_path(record)
-        "/madmin/#{model.model_name.collection}/#{record.id}/edit"
+        "/madmin/#{model.model_name.collection}/#{friendly_id(record)}/edit"
```

Comment:
I think the paths just need to be modify to use `to_param` so they will render either the ID or the friendly ID.

```ruby
"/madmin/#{model.model_name.collection}/#{record.to_param}"
```

Reply:
Wow, I didn't know that! I have updated the code :)